### PR TITLE
add s2s and s2t client utility functions

### PR DESCRIPTION
**Pull request by rmittal-github**

### CODE REVIEW

**Comment by rmittal-github** [comment]
@PeganovAnton could you please review this PR? thanks.

File: riva/client/nmt.py [hunk 1]
```diff
@@ -25,6 +37,99 @@ def __init__(self, auth: Auth) -> None:
...
+            ``StreamingTranslateSpeechToSpeechResponse``
+            message `here
+            <https://docs.nvidia.com/deeplearning/riva/user-guide/docs/reference/protos/protos.html#riva-proto-riva-nmt-proto>`_.
+        """
+        generator = streaming_s2s_request_generator(audio_chunks, streaming_config)
+        for response in self.stub.StreamingTranslateSpeechToSpeech(generator, metadata=self.auth.get_auth_metadata()):
+            yield response
+
+
+    def streaming_s2t_response_generator(
```
Comment (virajkarandikar) [inline riva/client/nmt.py:87]: is this called anywhere?

File: riva/client/nmt.py [hunk 2]
```diff
@@ -25,6 +37,99 @@ def __init__(self, auth: Auth) -> None:
         self.auth = auth
         self.stub = riva_nmt_srv.RivaTranslationStub(self.auth.channel)
 
+    def streaming_s2s_response_generator(
```
Comment (virajkarandikar) [inline riva/client/nmt.py:40]: Is this called somewhere?

Comment (virajkarandikar) [reply riva/client/nmt.py:40]: I mean how are these utility functions getting tested?

File: riva/client/nmt.py (hunk 1 above)
Comment (rmittal-github) [reply riva/client/nmt.py:87]: yes, it is called from the notebook https://github.com/nvidia-riva/tutorials/pull/145/files

File: riva/client/nmt.py (hunk 2 above)
Comment (rmittal-github) [reply riva/client/nmt.py:40]: called from the notebook https://github.com/nvidia-riva/tutorials/pull/145/files

Comment (virajkarandikar) [reply riva/client/nmt.py:40]: ok.

**Review by virajkarandikar — APPROVED** [review]

**Pull request merged**